### PR TITLE
Followup of pr33

### PR DESCRIPTION
**Pull request by qiujian16**
1. fix some typos
2. change forceRollOut func to be immutable

### CODE REVIEW

File: pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go [hunk 1]
```diff
@@ -288,31 +290,25 @@ func (n *klusterletController) sync(ctx context.Context, controllerContext facto
...
-	if *forceRollOut {
-		return
-	}
-
+func (n *klusterletController) forceRollOutWorkAgent(ctx context.Context, config klusterletConfig) bool {
 	workDeploymentName := fmt.Sprintf("%s-work-agent", config.KlusterletName)
 	workDeployment, err := n.kubeClient.AppsV1().Deployments(config.KlusterletNamespace).Get(ctx, workDeploymentName, metav1.GetOptions{})
 	if err != nil {
-		return
+		return false
```
Comment (pmorie) [inline pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go:298]: shouldn't this be true? If we didn't find it, it should be rolled out, right?

**Review by pmorie — COMMENTED** [review]
One question

File: pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go (hunk 1 above)
Comment (qiujian16) [reply pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go:298]: I think we will create it in applyDeployment  if it is not found in the following. I will add a comment here.

**Pull request merged**